src/main/java/com/checkmarx/flow/service/FlowService.java

```diff
@@ -162,24 +162,27 @@ public ScanDetails executeCxScan(ScanRequest request, File cxFile) throws Machin
         }catch (CheckmarxException | GitAPIException e){
             log.error(ExceptionUtils.getMessage(e), e);
             Thread.currentThread().interrupt();
-            logRequest(request, scanId, cxFile, Status.FAILURE.build(e.getMessage()));
+            OperationResult scanCreationFailure = new OperationResult(OperationStatus.FAILURE, e.getMessage());
+            logRequest(request, scanId, cxFile, scanCreationFailure);
             throw new MachinaException("Checkmarx Error Occurred");
```

Comment:
In case of a failure it will log via the logRequest, but don't we want to supply the exception error message also to the thrown exception?

Reply:
fixed